var/spack/repos/builtin/packages/rust-bootstrap/package.py

```diff
@@ -23,6 +23,17 @@ class RustBootstrap(Package):
     # should update these binary releases as bootstrapping requirements are
     # modified by new releases of Rust.
     rust_releases = {
+        "1.81.0": {
```

Comment:
@alecbcs Can you review this?

Reply:
@ashim-mahara can you please break out `rust` and `rust-bootstrap` into their own PR?